hermes/probe/create/create.go

```diff
@@ -168,25 +167,26 @@ func CreateFile(ctx context.Context, target *probe.Target, fileID int32, fileSiz
 	target.LatencyMetrics.APICallLatency[metrics.APICreateFile][status].Metric(hermesAPILatencySeconds).AddFloat64(time.Now().Sub(start).Seconds())
 
 	// Verify that the file that has just been created is in fact present in the target system
-	query := &storage.Query{Prefix: fmt.Sprintf(FileNameFormat, fileID, "")}
+	fileNamePrefix := fmt.Sprintf(FileNameFormat, fileID, "")
+	query := &storage.Query{Prefix: fileNamePrefix}
 	start = time.Now()
 	objIter := client.Bucket(bucketName).Objects(ctx, query)
-	finish := time.Now()
 	obj, err := objIter.Next()
```

Comment:
Remove this and make it the first action in the for loop.
Follow the pattern in https://godoc.org/cloud.google.com/go/storage#hdr-Listing_objects.

Reply:
noted

Reply:
changed